src/ui/window/BrowserWindow.qml

```diff
@@ -262,10 +226,9 @@ FluidWindow {
         }
 
         MenuItem {
-            text: "Search in page"
+            text: "Find in page"
```

Comment:
@plfiorini Yeah, all strings are untranslatable right now, I will create a separate issue for that.